File: лаба 1.cpp
#include <iostream>
using namespace std;

int main()
{
	// Визначити, скільки повних годин та повних хвилин пройшло з початку доби до k-ої її секунди
	setlocale(0, "rus");

	cout << "Enter the number of seconds: ";
	int time_sec;	// число секунд k, що пройшли з початку дня
	cin >> time_sec;	// вхідні дані

	int time_min = time_sec / 60;	// визначення к-ть повних хвилин з к-ті секунд
	cout << "Minutes: " << time_min << endl;	// вихідні дані

	int time_hour = time_min / 60;	// визначення к-ть повних годин з к-ті хвилин
	cout << "Hours: " << time_hour << endl;	// вихідні дані

	system("Pause");
}

File: лаба 1.py
#Визначити, скільки повних годин та повних хвилин пройшло з початку доби до k-ої її секунди

time_sec = int(input("Enter the number of seconds: "))  # ведення числа секунд k, що пройшли з початку дня

time_min = int(time_sec / 60)   # визначення к-ть повних хвилин з к-ті секунд
print("Minutes: ", time_min)    # вихідні дані

time_hour = int(time_min / 60)  # визначення к-ть повних годин з к-ті хвилин
print("Hours: ", time_hour)     # вихідні дані


